src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/fsm/SwitchSyncFsm.java

```diff
@@ -171,15 +171,15 @@ public String getKey() {
 
     protected void initialized(SwitchSyncState from, SwitchSyncState to,
                                SwitchSyncEvent event, Object context) {
-        log.info("Key: {}, sync FSM initialized", key);
+        log.info("The switch sync process for {} has been started (key={})", switchId, key);
```

Comment:
just for the sake of curiosity, is it better to move key from the beginning from the search perspective?

Reply:
it was aligned at least as for me. yeah, why not. (not a blocker, just curious)